apis/billing/v1alpha1/billingaccount_types.go

```diff
@@ -45,7 +45,7 @@ type BillingAccountSpec struct {
 	CurrencyCode *string `json:"currencyCode,omitempty"`
 
 	// Optional. The billing account's parent resource.
-	ParentBillingAccountRef *BillingAccountRef `json:"parent,omitempty"`
+	ParentRef *BillingAccountRef `json:"parentRef,omitempty"`
```

Comment:
hmmm, I thought we would prefer the <ParentKind>Ref rather than `parentRef` for its usage simplicity: users don't need to know which parent it is, just use the kind to find the parent. Also, Apigee uses <ParentKind>Ref (maybe a pattern to follow?)

A different question: I just realize this parent Kind is the same as the child kind (BillingAccount). Is that right? If so, maybe`parentRef` is good choice for this specific case.

Reply:
Good point.  And yes, this is a recursive reference, so maybe parentRef is fine.  If not parentRef, then I think it should be parentBillingAccountRef